sdcm/cluster_cloud.py

```diff
@@ -200,6 +207,10 @@ def start_coredump_thread(self):
     def is_cloud() -> bool:
         return True
 
+    @cached_property
+    def cql_address(self):
```

Comment:
This gonna be trickery for testing from a local machine, we should test if we can still run from local machine after those chamges